.travis.yml

```diff
@@ -29,6 +29,11 @@ jobs:
     - make test CHARTS=enterprise-suite-latest
 
   - name: ES Frontend E2E
+    addons:
+    apt:
```

Comment:
Doesn't the `apt` block need to be indented?

Reply:
good catch!